Makefile

```
@@ -82,6 +82,15 @@ test-failpoint:
 	BBOLT_VERIFY=all TEST_FREELIST_TYPE=array go test -v ${TESTFLAGS} -timeout 30m ./tests/failpoint
 
 .PHONY: test-robustness # Running robustness tests requires root permission
-test-robustness:
+test-robustness: check-gofail
```

We don't need to execute `make gofail-enable` in workflow
```suggestion
test-robustness: gofail-enable
```

---

The bbolt is built after `make gofail-enable`. That's why I add new recipe.

---

If we made a new Makefile recipe for `make build` we could have like below and they would execute in that sequence right?

```suggestion
test-robustness: gofail-enable build
```

That way the user wouldn't need to do anything manually?

---

Sounds good. I was thinking that bbolt is binary helper and we don't need bind two things together.
Let me try to add build recipe.